source/index.html.md

```diff
@@ -964,23 +964,6 @@ Examples:
 }
 ````
 
-### Stop recurring status
-
-On any recurring transaction, issuers can respond with <i>stop recurring</i>,
-withdrawing the consent from the cardholder for a merchant to initiate
-transactions on their behalf.
-
-A <i>stop recurring</i> status is both a decline of the current authorization,
-but also a notice to halt all future recurring transactions on this card from
-this merchant.
-
-If <i>stop recurring</i> responses are not adhered to and recurring
-transactions on this card continues, a fee pr. transaction is incurred by
-the merchant from the card scheme.
-
-Merchants should contact the cardholder to resolve this issue. Only after the
-withdrawal of consent has been lifted, can recurring transactions be resumed.
-
```

Comment:
Couldn't *some* of this be considered protocol and spared? I'm specifically thinking about the two first paragraphs detailing the intent. In other words: true and accurate for all schemes/issuers regardless of contracts.

Reply:
Only including the first 2 paragraph leaves the obvious question.

*What can I do to "unban" my merchant?*

The last paragraph could be included to calm the nerves a bit

>On any recurring transaction, issuers can respond with stop recurring, withdrawing the consent from the cardholder for a merchant to initiate transactions on their behalf.
>
>A stop recurring status is both a decline of the current authorization, but also a notice to halt all future recurring transactions on this card from this merchant.
>
>Merchants should contact the cardholder to resolve this issue. Only after the withdrawal of consent has been lifted, can recurring transactions be resumed.

Reply:
You're right.

I can't get the wording right, but I'd rather go with something along the lines of: this is a permanent condition of the card which requires an action from the issuer or cardholder to change.

The original one is probably OK as well.

Reply:
I have tried to make the section even less opinionated.
But kept emphasis on recurring are disallow until `stop recurring` is lifted.